src/Common/Entity/Program.php

```diff
@@ -311,4 +312,20 @@ public function setEndDate($end_date): void
     {
         $this->end_date = $end_date;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getEnableEmailLogin()
```

Comment:
it may be convenient to have enableEmailLogin() and disableEmailLogin() functions that do the work so the consumer of this SDK does not need to know it requires an integer.